pylintrc

```diff
@@ -335,19 +326,19 @@ max-args=20
 
 # Argument names that match this expression will be ignored. Default to name
 # with leading underscore
-ignored-argument-names=_.*
+ignored-argument-names=^_
```

Comment:
Sure? Wouldn't we like to keep an option to have "undocumented" arguments that may be removed in the future?